tests/mock/clitools.rs

```diff
@@ -99,9 +99,9 @@ pub fn setup(s: Scenario, f: &dyn Fn(&mut Config)) {
     env::remove_var("RUSTUP_TOOLCHAIN");
     env::remove_var("SHELL");
     env::remove_var("ZDOTDIR");
-    // clap does it's own terminal colour probing, and that isn't
+    // clap does its own terminal colour probing, and that isn't
     // trait-controllable, but it does honour the terminal. To avoid testing
-    // claps code, lie about whatever terminal this process was started under.
+    // clap's code, lie about whatever terminal this process was started under.
```

Comment:
While these fixes are correct, they seem a little odd in this PR :D

Reply:
Yeah, I noticed them while I was working on some code, and it's sometimes annoying to open a whole new PR just to fix two typos in a comment :)